### PR TITLE
Remove onAccountSet from UploadFilesActivity

### PR DESCRIPTION
**Pull request by ezaquarii**
Remove onAccountSet from UploadFilesActivity

Signed-off-by: Chris Narkiewicz <hello@ezaquarii.com>

### CODE REVIEW

File: src/main/java/com/owncloud/android/ui/activity/UploadFilesActivity.java [hunk 1]
```diff
@@ -677,8 +677,8 @@ public void onCancel(String callerTag) {
     }
 
     @Override
-    protected void onAccountSet(boolean stateWasRecovered) {
```
Comment (ezaquarii) [inline src/main/java/com/owncloud/android/ui/activity/UploadFilesActivity.java]: This method is called from `onStart` if account is set and `stateWasRecovered` is not used. There is no flow control change here.

**Comment by ezaquarii** [comment]
This PR can be reviewed and testes. Mege requires merging #4677 first.

**Comment by tobiasKaminsky** [comment]
I rebased it, so that this PR only contains real change.
Tested, works :+1:

**Comment by ezaquarii** [comment]
@tobiasKaminsky Ready for merge

**Pull request merged**